mobile/src/screens/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,24 +1,66 @@
 import React from 'react';
 
-import { Button, Link, Text, View } from '#libs/components/components';
+import {
+  Button,
+  Input,
+  Label,
+  Link,
+  ScrollView,
+  Text,
+} from '#libs/components/components';
 import { RootScreenName } from '#libs/enums/enums';
+import { useAppForm, useCallback } from '#libs/hooks/hooks';
+import { type UserSignInRequestDto } from '#packages/users/users';
+import { userSignInValidationSchema } from '#packages/users/users';
+
+import { USER_SIGN_IN_DEFAULT_VALUE } from './libs/constants';
+import { styles } from './styles';
 
 type Properties = {
-  onSubmit: () => void;
+  onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => {
+const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
+  const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
+    defaultValues: USER_SIGN_IN_DEFAULT_VALUE,
+    validationSchema: userSignInValidationSchema,
+  });
+
+  const handleFormSubmit = useCallback((): void => {
+    void handleSubmit(onSubmit)();
+  }, [handleSubmit, onSubmit]);
+
   return (
-    <View>
-      <Text>Sign In</Text>
-      <Button
-        label="Sign in"
-        onPress={(): void => {
-          // TODO: handle Sign in
-        }}
-      />
-      <Link label="Go to Sign Up" to={`/${RootScreenName.SIGN_UP}`} />
-    </View>
+    <React.Fragment>
+      <ScrollView style={styles.formContainer}>
+        <Label />
+        <Text style={styles.titleText}>Sign In to your account</Text>
+        <Input
+          control={control}
+          errors={errors}
+          label="Email"
+          name="email"
+          placeholder="Enter your email"
+        />
+        <Input
+          control={control}
+          errors={errors}
+          label="Password"
+          name="password"
+          placeholder="Enter your password"
+        />
+        <Button label="Sign in" onPress={handleFormSubmit} />
+        <Link
+          label={
+            <React.Fragment>
+              Don&apos;t have an account? Go to{' '}
+              <Text style={{ fontWeight: 'bold' }}>Sign Up</Text>
+            </React.Fragment>
+          }
+          to={`/${RootScreenName.SIGN_UP}`}
```

Comment:
```suggestion
          to={RootScreenName.SIGN_UP}
```

Reply:
In this case I get an error: ERROR  Error: The path must start with '/' (Sign Up)., so I left everything as it was